src/network/index.js

```diff
@@ -2413,29 +2413,26 @@ realityEditor.network.postAiApiKeys = function(endpoint, azureApiKey, isInit = f
         });
 }
 
-realityEditor.network.postQuestionToAI = function(conversation, extra) {
-    let worldId = realityEditor.worldObjects.getBestWorldObject();
-    let ip = worldId.ip;
-    let port = realityEditor.network.getPort(worldId);
+realityEditor.network.postQuestionToAI = async function(conversation, extra) {
     let route = '/ai/question';
     
-    this.postData(realityEditor.network.getURL(ip, port, route), 
-        {
+    const response = await fetch(window.location.origin + route, {
+        headers: {
+            'Accept': 'application/json',
```

Comment:
This will work with the code I pointed out in cloud-proxy if you add
Authorization: `Bearer ${window.localStorage.something}`